pkg/apiserver/controllers/v1/decisions.go

```diff
@@ -93,11 +93,17 @@ func (c *Controller) DeleteDecisionById(gctx *gin.Context) {
 		gctx.JSON(http.StatusBadRequest, gin.H{"message": "decision_id must be valid integer"})
 		return
 	}
-	nbDeleted, err := c.DBClient.SoftDeleteDecisionByID(decisionID)
+	nbDeleted, deletedFromDB, err := c.DBClient.SoftDeleteDecisionByID(decisionID)
 	if err != nil {
 		c.HandleDBErrors(gctx, err)
 		return
 	}
+	//transform deleted decisions to be sendable to capi
+	deletedDecisions, err := FormatDecisions(deletedFromDB, false)
+	if err != nil {
+		log.Warningf("failed to format decisions: %v", err)
+	}
+	c.DecisionDeleteChan <- deletedDecisions
```

Comment:
do we want to wait for this operation since the channel is not buffered

Reply:
On the other end of the chan, we have a routine dedicated to polling the chan and building a cache `SyncDecisions`.
We can throw this in a tombless go-routine maybe, wdyt ?